tests/bears/BearTest.py

```diff
@@ -1,10 +1,14 @@
 import multiprocessing
+import shutil
 import unittest
+
 from os.path import abspath, exists, isfile, join, getmtime
-import requests_mock
-import shutil
 from time import sleep
 
+import requests
+
+import requests_mock
```

Comment:
For test modules, I like to keep functionality (i.e. requests) vs test components (requests-mock) in separate groups.